app-i18n/fbterm/fbterm-1.7_p20190503.ebuild

```
@@ -0,0 +1,67 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI="7"
+
+inherit autotools fcaps toolchain-funcs
+
+DESCRIPTION="Fast terminal emulator for the Linux framebuffer"
+HOMEPAGE="https://github.com/gjedeer/fbterm"
+
+MY_COMMIT="ccea326dd73f4d6b6442fde7ba7c2be9cd35c6df"
+SRC_URI="https://github.com/gjedeer/fbterm/archive/${MY_COMMIT}.tar.gz -> ${P}.tar.gz"
+S="${WORKDIR}/fbterm-${MY_COMMIT}"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~arm ~arm64 ~x86"
+IUSE="gpm video_cards_vesa"
+
+DEPEND="
+	media-libs/fontconfig
+	media-libs/freetype:2
+	>=sys-libs/ncurses-6.1
+	gpm? ( sys-libs/gpm )
+	video_cards_vesa? ( dev-libs/libx86 )
+"
+BDEPEND="
+	virtual/pkgconfig
+"
```

I'd prefer no newline after or before double quote.

```suggestion
DEPEND="media-libs/fontconfig
	media-libs/freetype:2
	>=sys-libs/ncurses-6.1
	gpm? ( sys-libs/gpm )
	video_cards_vesa? ( dev-libs/libx86 )"
BDEPEND="virtual/pkgconfig"
```

---

Done.

---

Ebuild is missing:
```
RDEPEND="${DEPEND}"
```